client/me/notification-settings/reader-subscriptions/index.jsx

```diff
@@ -299,15 +303,14 @@ class NotificationSubscriptions extends Component {
 							</FormFieldset>
 						) }
 
-						<FormButton
-							isSubmitting={ this.props.isUpdatingUserSettings }
+						<Button
+							variant="primary"
 							disabled={ this.props.isUpdatingUserSettings || ! this.props.hasUnsavedUserSettings }
-							onClick={ this.handleClickEvent( 'Save Notification Settings Button' ) }
+							isBusy={ this.props.isUpdatingUserSettings }
+							onClick={ this.handleSubmitButtonClick }
 						>
-							{ this.props.isUpdatingUserSettings
-								? this.props.translate( 'Saving…' )
-								: this.props.translate( 'Save notification settings' ) }
-						</FormButton>
+							{ this.props.translate( 'Save notification settings' ) }
```

Comment:
Should we consistently say "Save settings" as we say on other tabs?

<img width="1097" alt="Screenshot 2025-05-12 at 17 32 54" src="https://github.com/user-attachments/assets/63c674ab-86b3-415c-9fb7-a5a159bd5adf" />
<img width="1097" alt="Screenshot 2025-05-12 at 17 32 49" src="https://github.com/user-attachments/assets/751020ba-5962-491f-ac2d-bd2ef7c2b801" />